.vscode/tasks.json

```diff
@@ -34,6 +34,19 @@
         ],
         "problemMatcher": "$msCompile",
         "group": "build"
+    },
+    {
+        "label": "build directory skip analyzers",
+        "command": "dotnet",
+        "type": "shell",
+        "args": [
+          "msbuild",
+          "-p:UseRoslynAnalyzers=false",
+          "-p:GenerateFullPaths=true",
+          "${fileDirname}"
+        ],
+        "problemMatcher": "$msCompile",
+        "group": "build"
```

Comment:
This makes it easier to build in VS Code. Now you can build the "current project" by running `dotnet build` on the directory of the open file.